src/app/content/highlights/components/Card.tsx

```diff
@@ -4,101 +4,125 @@ import { HTMLElement } from '@openstax/types/lib.dom';
 import flow from 'lodash/fp/flow';
 import React from 'react';
 import { connect, useSelector } from 'react-redux';
-import styled, { css } from 'styled-components/macro';
+import styled from 'styled-components';
 import * as selectAuth from '../../../auth/selectors';
 import { User } from '../../../auth/types';
-import { findElementSelfOrParent, scrollIntoView } from '../../../domUtils';
-import theme from '../../../theme';
 import { AppState, Dispatch } from '../../../types';
-import { assertWindow, remsToEms } from '../../../utils';
-import { contentTextWidth, searchResultsBarDesktopWidth, sidebarDesktopWidth } from '../../components/constants';
-import { disablePrint } from '../../components/utils/disablePrint';
-import { styleWhenSidebarClosed } from '../../components/utils/sidebar';
 import * as selectHighlights from '../../highlights/selectors';
 import * as selectSearch from '../../search/selectors';
 import * as selectContent from '../../selectors';
 import * as contentSelect from '../../selectors';
 import { stripIdVersion } from '../../utils/idUtils';
-import { clearFocusedHighlight, createHighlight, deleteHighlight, updateHighlight } from '../actions';
-import {
-  cardContentMargin,
-  cardFocusedContentMargin,
-  cardMinWindowMargin,
-  cardPadding,
-  cardWidth,
-  highlightStyles
-} from '../constants';
+import { clearFocusedHighlight, createHighlight, deleteHighlight, focusHighlight, updateHighlight } from '../actions';
+import { highlightStyles } from '../constants';
 import { HighlightData } from '../types';
 import { getHighlightLocationFilterForPage } from '../utils';
+import { mainCardStyles } from './cardStyles';
 import DisplayNote from './DisplayNote';
 import EditCard from './EditCard';
-import { cardBorder } from './style';
 
-interface Props {
+export interface CardProps {
   page: ReturnType<typeof selectContent['bookAndPage']>['page'];
   book: ReturnType<typeof selectContent['bookAndPage']>['book'];
   container?: HTMLElement;
   isFocused: boolean;
+  isTocOpen: boolean;
+  hasQuery: boolean;
   user: User;
   loginLink: string;
   highlighter: Highlighter;
   highlight: Highlight;
   create: typeof createHighlight;
+  focus: typeof focusHighlight;
   save: typeof updateHighlight;
   remove: typeof deleteHighlight;
   blur: typeof clearFocusedHighlight;
   data?: HighlightData;
   className: string;
+  zIndex: number;
+  topOffset?: number;
+  onHeightChange: (ref: React.RefObject<HTMLElement>) => void;
+  onFocus: () => void;
+  onBlur: () => void;
 }
 
 // tslint:disable-next-line:variable-name
-const Card = (props: Props) => {
-  const annotation = props.data && props.data.annotation;
+const Card = ({
+  book,
+  blur,
+  className,
+  create,
+  data,
+  focus,
+  highlight,
+  highlighter,
+  loginLink,
+  isFocused,
+  onFocus,
+  onBlur,
+  onHeightChange,
+  page,
+  remove,
+  save,
+  user,
```

Comment:
i would prefer to use `props.` repeatedly rather than use up 20 lines of space on this destructuring